api/notes.md

```diff
@@ -0,0 +1,16 @@
+Checks toggles via a call to CheckFeatureIsEnabled which is in the service.
```

Comment:
Typically we can call this README.md in the specific folder.

Reply:
Ah yes, this was just meant for me but I'll update it and turn it into a readme.